docs/topics/db/models.txt

```diff
@@ -533,9 +533,11 @@ There are a few restrictions on the intermediate model:
 * For a model which has a many-to-many relationship to itself through an
   intermediary model, two foreign keys to the same model are permitted, but
   they will be treated as the two (different) sides of the many-to-many
-  relationship. If there are *more* than two foreign keys though, you
-  must also specify ``through_fields`` as above, or a validation error
-  will be raised.
+  relationship. The first foreign key in the intermediary model will be taken
```

Comment:
Also similar question as to whether saying, "If ``through_fields`` is not specified, the first foreign key..."